OpenTabletDriver.Desktop/AppInfo.cs

```diff
@@ -35,13 +35,13 @@ public static AppInfo Current
                 PluginPlatform.Linux => new AppInfo
                 {
                     ConfigurationDirectory = GetExistingPath("$XDG_DATA_HOME/OpenTabletDriver/Configurations", "~/.local/share/OpenTabletDriver/Configurations"),
-                    AppDataDirectory = GetPath("$XDG_CONFIG_HOME/OpenTabletDriver", "~/.config/OpenTabletDriver"),
+                    AppDataDirectory = GetPath(Path.Join(ProgramDirectory, "userdata"), "$XDG_CONFIG_HOME/OpenTabletDriver", "~/.config/OpenTabletDriver"),
```

Comment:
You need to use `GetExistingPathOrLast` instead of `GetPath` or it will find the `userdata` folder always and create it if it doesn't exist instead of only using the userdata folder when it already exists.

```suggestion
                    AppDataDirectory = GetExistingPathOrLast(Path.Join(ProgramDirectory, "userdata"), "$XDG_CONFIG_HOME/OpenTabletDriver", "~/.config/OpenTabletDriver"),
```

Reply:
right